src/layouts/ResourceRoom/ResourceRoom.tsx

```diff
@@ -116,10 +116,8 @@ const EmptyResourceRoom = () => {
           >
             <Input
               maxW="50%"
-              {...register("newDirectoryName", {
-                required:
-                  "Please ensure that you have entered a resource room name!",
-              })}
+              // eslint-disable-next-line react/jsx-props-no-spreading
+              {...register("newDirectoryName", { required: true })}
```

Comment:
i think this has different functionality - the previous one has the error message (the string) but this does not. was this intended?

Reply:
good catch - have re-included this error message! adc5ecf